test/Interop/Cxx/static/Inputs/static-local-var.h

```diff
@@ -0,0 +1,4 @@
+static inline int counter() {
```

Comment:
I don't think this function should be `static` -- that would make different TUs that use this function define separate copies of the `c` variable.

Reply:
Hups, yeah, fixed by not making the function static and not making it inline (and adding a .cc file with the definition).